mobile/Droidlet/Module/ArCore/DroidletARViewContainer.swift

```diff
@@ -0,0 +1,116 @@
+import SwiftUI
+import ARKit
+import Combine
+
+struct DroidletARViewContainer: UIViewRepresentable {
+    @ObservedObject var viewModel: HomeViewModel
+    var settings: UserSettings = .shared
+
+    private var cancellable: AnyCancellable?
+
+    var rayCastResultValue: ARRaycastResult?
+    var visionRequests = [VNRequest]()
+    
+    let resnetModel: Resnet50 = {
```

Comment:
nit: lets add a comment here to download the model before, unless that's been documented elsewhere in a README ?